fmriprep/workflows/base.py

```diff
@@ -88,6 +85,7 @@ def init_fmriprep_wf(
             from collections import namedtuple, OrderedDict
             BIDSLayout = namedtuple('BIDSLayout', ['root'])
             from fmriprep.workflows.base import init_fmriprep_wf
+            from niworkflows.utils.spaces import Space, SpatialReferences
```

Comment:
```suggestion
            from fmriprep.utils.spaces import Space, SpacesManager
```

Reply:
Working on these RN - maybe best hold up until I push (within the next 20min)

Reply:
Sounds good. Sorry. This was a lot to read through, so it's taken me like 2 hours of intermittent concentration.

I think I have a grasp on most of the moving parts, and the good news is I think things are overall getting simpler.

Reply:
> This was a lot to read through, so it's taken me like 2 hours of intermittent concentration.

Totally, I'll tell you how I call this refactor in private :p

> I think things are overall getting simpler.

This is important, good to know :)